src/forest.h

```diff
@@ -79,6 +79,7 @@ class Forest
 #endif
   friend class TimeInterval;
   friend class TimeIntervalIterator;
+  friend class TMRCA;
```

Comment:
Is this needed for the actual algorithm, or can it be moved up to the test setup?